.github/workflows/macos.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     name: macOS x86-64 conda
     # To enable this workflow on a fork, comment out:
     if: github.repository == 'numpy/numpy'
-    runs-on: macos-latest
+    runs-on: macos-13
```

Comment:
specifying macos-13 means that it's definitely built using x86_64

Reply:
that is useful future-proofing indeed